README.md

```diff
@@ -31,3 +44,51 @@ imap_data_access.download("imap/mag/l0/2024/01/imap_mag_l0_raw_202040101_2024010
 # Upload a calibration file that exists locally
 imap_data_access.upload("imap/mag/calibration/test_calibration.txt")
 ```
+
+## Configuration
```

Comment:
I would actually prefer this section to be at the beginning, otherwise users might try to upload a file without first realizing that they need to have a directory structure/`DATA_DIR` env variable configured. But this could be a case of my brain just reproaching the situation different from yours -- we are both right and both wrong 😅

Reply:
I had this first before, then decided to change it! :) So I can see both ways the same as you. I was thinking people often just want to start using a tool, which is why I put the cli thing up front. I'm happy to move this back up though if you think that is better that way. I don't have a major preference.

Reply:
Yeah I can't say I have a major preference either, so I will leave it up to you. This is when it would be nice to have a product owner who can make these end-user decisions for us 😂

Reply:
I left it as-is. Happy to add more content / rearrange later in follow-up PRs too.